pages/profile-edit/profile-edit.scss

```diff
@@ -1,19 +1,65 @@
 .profile-edit-page {
+  h5.email {
+    &::after {
+      @include form-helper-text("(email won't be displayed or shared)");
```

Comment:
This is fine for now but last I checked, pseudoelement content isn’t accessible.

Reply:
Let's stick with the existing way of adding help text 
e.g.,

```jsx
user_bio: {
      type: `textarea`,
      label: (
        <div>
          Bio <em>(do not include any personal information)</em>
        </div>
      ),
     ...
```

You can give the `em` element a class name if you think you need to target and style the help text.

Reply:
welp, that's not good. Let me double check that.